test/scheduler/test.js

```diff
@@ -0,0 +1,48 @@
+const test = require('@ava/test');
+const exec = require('../helpers/exec');
```

Comment:
This should give you access to the internal status objects for each test that executed, hopefully ordered. So you shouldn't need to handle stdout. Apologies for not spotting this earlier.